configure.ac

```diff
@@ -54,11 +54,7 @@ AH_BOTTOM([
 
 #include <string.h>
 
-#if ZEND_BROKEN_SPRINTF
-int zend_sprintf(char *buffer, const char *format, ...);
-#else
-# define zend_sprintf sprintf
-#endif
+#define zend_sprintf sprintf
```

Comment:
I think this should be moved into a normal header, and combined with the win32 definition, which is the same.

Reply:
I think so too... It went through well according to some testings of mine. Refactored in the recent commit more.